OPHD/Technology/TechnologyCatalog.h

```diff
@@ -28,6 +28,4 @@ class TechnologyCatalog
 
 	void readCategories(NAS2D::Xml::XmlElement& node);
 	void readTechnologiesInCategory(const std::string& categoryName, NAS2D::Xml::XmlElement& category);
-	Technology readTechnology(NAS2D::Xml::XmlElement& technology);
-	void readEffects(NAS2D::Xml::XmlElement& effects, Technology& technology);

```

Comment:
I like the structure change from moving the helper methods to an unnamed namespace.

I wonder if something similar could be done for the remaining two methods. It seems like they should be rather similar. Potentially you could remove includes for the XML code from the header file, which means fewer transitive includes where `TechnologyCatalog` is used.

Reply:
I was thinking the same thing. Ultimately that's going to be the plan, pull the other two out and moved into the unnamed namespace and keep the class definition very simple and clean.